standard/structs.md

```diff
@@ -138,6 +140,51 @@ struct_member_declaration
 
 Except for the differences noted in [§16.4](structs.md#164-class-and-struct-differences), the descriptions of class members provided in [§15.3](classes.md#153-class-members) through [§15.12](classes.md#1512-static-constructors) apply to struct members as well.
 
+### §struct_readonly Readonly members
+
+An instance member definition that includes the `readonly` modifier has the following restrictions:
+
+- The member shall not modify the value of a direct instance member of the receiver.
```

Comment:
Possibly be explicit about "field or property"? Is it okay to call a property setter if that doesn't actually change any instance state?

Reply:
(I guess we should include field-like events in this too.)